alpine/Dockerfile

```diff
@@ -7,12 +7,12 @@ ENV rakudo_version=${rakudo_version}
 
 RUN buildDeps=' \
         bash \
-        gnupg \
-        perl \
-        perl-encode \
         gcc \
+        gnupg \
         libc-dev \
         make \
+        perl \
+        perl-encode \
```

Comment:
Is this really useful? Or just makes build marginally faster?

Reply:
I think this change (sorted alphabetically) makes it easier to see where to add build dependencies.

Reply:
I think the build speed is the same as before the change, because there is no change in the number of build dependencies.